dgraph/cmd/bulk/mapper.go

```diff
@@ -20,7 +20,7 @@ import (
 
 	farm "github.com/dgryski/go-farm"
 	"github.com/golang/glog"
-	"github.com/golang/snappy"
+	"github.com/klauspost/compress/snappy"
```

Comment:
we should move to s2 here. Badger already is using s2, we can align with that here.